integration_tests/src/main/python/limit_test.py

```diff
@@ -30,19 +30,45 @@ def test_simple_limit(data_gen):
 
 @allow_non_gpu('CollectLimitExec', 'GlobalLimitExec', 'ShuffleExchangeExec')
 @pytest.mark.skipif(is_before_spark_340(), reason='offset is introduced from Spark 3.4.0')
-def test_non_zero_offset_fallback():
-    def test_fn(spark):
-        unary_op_df(spark, int_gen, num_slices=1).createOrReplaceTempView("offset_tmp")
-        # `offset` is not exposed to Pyspark, so use the sql string.
-        return spark.sql("select * from offset_tmp limit 10 offset 1")
-
-    assert_gpu_fallback_collect(
-        # We need some processing after the limit to avoid a CollectLimitExec
-        lambda spark: test_fn(spark).repartition(1),
-        'GlobalLimitExec',
-        conf = {'spark.sql.execution.sortBeforeRepartition': 'false'})
-
-    assert_gpu_fallback_collect(
-        test_fn,
-        'CollectLimitExec',
-        conf = {'spark.sql.execution.sortBeforeRepartition': 'false'})
+@allow_non_gpu('CollectLimitExec', 'GlobalLimitExec', 'ShuffleExchangeExec')
```

Comment:
This line is a duplicate of line 31, and both should be removed after supporting `offset`.